common-util/api/nftDetails.js

```diff
@@ -50,7 +50,6 @@ export const getLatestMintedNft = memoize(async (account) => {
 
     return { details: null, tokenId: null };
   } catch (e) {
-    notifyError('Error fetching NFT details');
```

Comment:
this is confusing users, will think next week about better supporting of base/mainnet switching